libc/src/ckb_cell_fs.c

```diff
@@ -41,7 +52,7 @@ int get_file(const CellFileSystem *fs, const char *filename, FSFile **f) {
 
 int ckb_get_file(const char *filename, FSFile **file) { return get_file(CELL_FILE_SYSTEM, filename, file); }
 
-int load_fs(CellFileSystem **fs, void *buf, uint64_t buflen) {
+int load_fs(CellFileSystem **fs, const char *prefix, void *buf, uint64_t buflen) {
```

Comment:
check that `prefix` is a valid path, one of following:
1. non empty
2. start with `/`
3. can be possible end with `/`
4. not end with `/` e.g. `/aaa`

Reply:
I added a normalize function to handle these prefix.